apps/definition/mix.exs

```diff
@@ -24,7 +24,7 @@ defmodule Definition.MixProject do
   defp deps do
     [
       {:ok, in_umbrella: true},
-      {:norm, "~> 0.10.2"},
+      {:norm, "0.10.4"},
```

Comment:
Gee, two version increases?

Reply:
We should set this to `~> 0.10.0` and use `mix deps.update norm` to update, until we move to `0.11.x`.